packages/kuma-gui/src/app/services/views/ServiceDetailView.vue

```diff
@@ -18,53 +18,59 @@
       <div
         class="stack"
       >
-        <KCard>
-          <DataLoader
-            :src="uri(sources, `/meshes/:mesh/service-insights/:name`, {
-              mesh: route.params.mesh,
-              name: route.params.service,
-            })"
-            v-slot="{ data }"
+        <DataLoader
+          :src="uri(sources, `/meshes/:mesh/service-insights/:name`, {
+            mesh: route.params.mesh,
+            name: route.params.service,
+          })"
+          v-slot="{ data }"
+        >
+          <XAboutSection
+            :title="t('services.internal-service.about.title')"
+            :created="data.creationTime ? t('common.formats.datetime', { value: Date.parse(data.creationTime) }) : undefined"
+            :modified="data.modificationTime ? t('common.formats.datetime', { value: Date.parse(data.modificationTime)}) : undefined"
           >
```

Comment:
Things would be a lot easier to use if we didn't have to do this. Just to be clear ,I know this PR removes usage of `XTimespan`, but seeing as `XTimespan` only ever displays a span of two dates, the date formatting is contained inside the component, meaning less work at the call site and a more ergonomic component. This is why XTimespan is built as it is.

Actually that made me think, should we be deleting the XTimespan component in this PR or do you think we might need it to show dates somewhere else?

Reply:
Yeah, I fully agree and dislike that we need to call it like this. This should be handled by the component, also because of consistency, because it does not enforce a common date-time format in all the various places this component is used.

I quickly went through the code and found that I missed to refactor the `HostnameGeneratorDetailView` accordingly. This is the only place left where we use `XTimespan`. So, I guess we can remove it. Once we need it again, we can add it back.

Reply:
Removed `XTimespan`. For future reference: https://github.com/kumahq/kuma-gui/pull/3300/commits/2d98182a1323b9bd1b627f6af11fa65e5e93176e

Reply:
Oh, sorry this got lost in my mind 😅

Reply:
Well to be fair I had in my mind that I did this in XTimespan, but apparently not! 😆 Ah now I remember 🤦 thanks for that link!

https://github.com/kumahq/kuma-gui/blob/484769bc4ff0113b81aff9cd0c8f0eb164923022/packages/kuma-gui/src/app/x/components/x-timespan/README.md?plain=1#L15-L16

Umm, I guess that makes me still on the fence with this one 😆 , I'll leave it with you if you want to change then fine or you want to leave as is, lets resolve. I think apart from this theres only one more left.

Reply:
Ah 😄 anyways, I've reworked it and moved the formatting to the component.